.circleci/config.yml

```diff
@@ -1553,6 +1566,86 @@ jobs:
             - repo/HapiApp2.0
             - root/.m2
             - repo/hedera-node
+
+  build-platform-and-services:
+    parameters:
+      workflow-name:
+        type: string
+        default: ""
+    executor:
+      name: build-executor
+      workflow-name:  << parameters.workflow-name >>
+    steps:
+      - attach_workspace:
+          at: /
+      - add_ssh_keys:
+          fingerprints:
+            - "96:47:c4:5c:e7:45:06:c5:26:a5:85:ef:41:22:2f:d6"
+            - "14:21:e9:81:1f:ae:df:ec:11:60:4a:49:e0:b9:bb:58"
+            - "e7:a6:3e:34:3e:d8:fe:64:2c:7f:b6:57:45:03:44:ac"
+      - checkout
+      - run:
+          name: Build hedera-services repo
+          command: |
+                mvn --no-transfer-progress clean install -DskipTests
+      - run:
+          name: Checkout swirlds-platform repos and build
+          command: |
+            cd /; GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" \
+              git clone  --recurse-submodule ssh://git@github.com/swirlds/swirlds-platform.git
+            cd /swirlds-platform; git submodule update --init --recursive --checkout
+            cd /swirlds-platform/regression;   git checkout 334-D-Add-services-config
+
+      - run:
+          name: Install necessary tools
+          command: |
+            sudo apt update -y;
+            sudo apt install net-tools;
+            sudo apt install -y curl;
+            sudo apt install -y python3.7;
+            sudo rm /usr/bin/python3;
+            sudo ln -s python3.7 /usr/bin/python3;
+            sudo apt install python3-pip;
+            pip3 -q install matplotlib;
+            sudo apt-get install -y python3-setuptools;
+            pip3 install awscli;
+
+      - run:
+          name: Save PEM file to keys folder
+          command: |
+            cp ~/.ssh/id_rsa_e7a63e343ed8fe642c7fb657450344ac /swirlds-platform/regression/keys/services-regression.pem
+
+      - run:
+          name: Run various suites in SuiteRunner
+          command: |
+            cd /swirlds-platform/regression; ./regression_services.sh AwsNightlyServicesRegressionCfg.json /repo
+
+      - run:
+          name: Performance Regression
+          command: |
+            cd /swirlds-platform/regression; ./regression_services.sh AwsServicesLoadTestRegressionCfg.json /repo
+
+      - run:
+          name: Run Network Error regression tests
+          command: |
+            cd /swirlds-platform/regression; ./regression_services.sh AwsSerivceRegressionNetworkError.json /repo
+
+      - run:
+          name: Sync to AWS
+          command: |
+            aws s3 sync /swirlds-platform/regression/results/ s3://hedera-service-regression-jrs
+
+      - persist_to_workspace:
+          root: /
+          paths:
+            - repo/.git
+            - repo/.circleci
+            - repo/hapiProto
+            - repo/test-clients
+            - repo/HapiApp2.0
+            - root/.m2
+            - repo/hedera-node
```

Comment:
Since there is no following job depends on the workspace, maybe we don't need this 'persist_to_workspace' step any more.

Reply:
Fixed. Thank you!!